code/game/machinery/pipe/construction.dm

```diff
@@ -51,40 +53,54 @@ Buildable meters
 	..()
 	to_chat(user, "<span class='notice'>Alt-click to rotate it clockwise.</span>")
 
-/obj/item/pipe/New(loc, pipe_type, dir, obj/machinery/atmospherics/make_from)
+/obj/item/pipe/New(loc, _pipe_type, _dir, obj/machinery/atmospherics/make_from)

```

Comment:
shouldn't initialize be used or is there some reason why it shouldn't

Reply:
Nope legacy code I'll get through it a bit.